.azure-pipelines/generation-templates/php-beta-migration.yml

```diff
@@ -0,0 +1,9 @@
+steps:
+  - pwsh: '$(scriptsDirectory)/copy-php-beta-migration-models.ps1'
```

Comment:
reuse https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/blob/dev/.azure-pipelines/generation-templates/php-beta.yml instead and make paths template parameters